### ISSUE TITLE
dev environnement is inusable

### ISSUE THREAD

**Issue by jelti** [Sylius/Sylius#3777]
Hello guys,
i've just installed a fresh version of sylius/sylius-standard (using composer -s dev)
The problem is that the performance of the dev env are really bad ( ive created a test product and need to wait 20 sec to get is displayed)
i understand that in the dev env, the cache is not really used to avoid cache problems but i thing that there is another issue in my installation
i'm using windows 8, with XAMP (apache2) and the sylius project is installed locally on my laptop (SSD disk)

you can check the performances logs in the file attached

Thank you very much for your help
M JELTI

http://localhost/app_dev.php/p/produit-test
![file sylius](https://cloud.githubusercontent.com/assets/11663571/12006665/4ced40c6-abe0-11e5-9b91-5b4ff52b7d7f.png)

**Comment by tristanbes** [comment]
Sylius doesn't work well on windows there are few issues like your regarding windows and performance. They'll improve windows support over time but clearly not a priority for now.

TLDR: don't use sylius with windows :)

**Comment by jelti** [comment]
Hello, 
Thank you very much for your answer
do you know witch os/apache ive to install to get the best performance?
thank you very much
M. JELTI

**Comment by tristanbes** [comment]
any linux based, something like Unbutu and you won't have problems

**Comment by crazy-max** [comment]
For me it takes 15 / 20 seconds on Neard 1.0.17, and the same on Xampp, Wamp or Winginx.
I have tried many things like :
* FastCGI / FPM
* Disable Xdebug
* RamDisk
* NFS share

But nothing worked :(

I have made cachegrind test :

![image1](https://cloud.githubusercontent.com/assets/1951866/12756053/438b2ce0-c9d2-11e5-8aef-d3997910d070.png)

and try to modify realpath configuration with `realpath_cache_size = 4M` and no gain..
